src/AccountDropdown.tsx

```diff
@@ -32,6 +33,7 @@ export const AccountDropdown: React.FC = (props) => {
   });
 
   const logout = () => {
+    localStorage.setItem('mcart', createCartIdentifier());
```

Comment:
@petro97 @yulia-dnistrian I just noticed this. It seems a bit overkill. We need to import moltin/request into the project only for this call? Is this not supported in our SDK?

Reply:
@shaunmaharaj I agree, It seems overkill. I removed @moltin/request